zndraw/server/events.py

```diff
@@ -10,6 +10,34 @@
 log = logging.getLogger(__name__)
 
 
+def _webclients_room(data: dict) -> str:
+    """Return the room name for the webclients."""
+    if "sid" in data:
+        return data["sid"]
+    return f"webclients_{data['token']}"
+
+
+def _webclients_default(data: dict) -> str:
+    """Return the SID of the default webclient."""
+    if "sid" in data:
+        return data["sid"]
+    # TODO: if there is a keyerror, it will not be properly handled and the
+    #  python interface is doomed to wait for TimeoutError.
+    return app.config["ROOM_HOSTS"][data["token"]][0]
```

Comment:
Then do a `try...except` here?

Reply:
I want to handle this in a different PR, but will add `try..except` for now - see https://github.com/zincware/ZnDraw/issues/332